examples/postprocess_dag.json

```diff
@@ -55,15 +57,28 @@
       "additional_attributes": {
         "nt_db": "s3://idseq-database/20170824/blast_db/nt"
       }
+    },
+    {
+      "in": ["fastqs"],
```

Comment:
can we move this to a separate dag since it's a standalone?

Reply:
@yunfangjuan Would make so we can be completely decoupled from the rest of the pipeline. On the other hand, I feel having to make a completely new run_stage for this new step would be too much overhead? Would prefer to piggy-back on postprocess DAG if you think it's safe

Reply:
I thought you were going to remove the step in the post_process example. This doesn't have anything to do with the actual web integration. If you want to put this somewhere. I would actually suggest to put this in the host_filtering.json if this step is not resource intensive.

Reply:
As noted by Yun-Fang, please remove any changes to this file. Conceptually everything is captured in `examples/amr_id_dag.json`. And for testing, it is best to just use amr_id_dag, rather than bothering with all of the other postprocessing steps (since there are actually no file dependencies linking AMR step to other postprocessing steps right now).

Reply:
Sure, I can do that. Since we decided to move this step to host filtering, and since it does share the same input files as the STAR step (i.e. the fastqs), shall I add to the hostfiltering.json dag? So I can ensure that my code works even when two steps share the same input. I can also stick with just the amr_id_dag if that's the best option.